azure-pipelines.yml

```diff
@@ -270,6 +270,43 @@ jobs:
           testRunTitle: 'Python $(python.version), image $(imageName)'
         condition: succeededOrFailed()
 
+      - task: PublishCodeCoverageResults@1
+        displayName: 'Publish Code Coverage Results'
+        inputs:
+          codeCoverageTool: Cobertura
+          summaryFileLocation: '$(System.DefaultWorkingDirectory)/**/coverage.xml'
+
+- template: azure-pipelines-steps.yml
+  parameters:
+    package: '-e .[tf,plt]'
+    job:
+      job: Tests_other
+      dependsOn: 'EvalChanges'
+      condition: eq(dependencies.EvalChanges.outputs['output.testCode'], 'True')
+      displayName: 'Run tests (Other)'
+      steps:
+      - script: 'pip install pytest pytest-runner && python setup.py pytest'
+        displayName: 'Unit tests'
+        env:
+          PYTEST_ADDOPTS: '-m "cate_api" -n auto'
+          COVERAGE_PROCESS_START: 'setup.cfg'
+      - script: 'python setup.py pytest'
+        displayName: 'Unit tests'
+        env:
+          PYTEST_ADDOPTS: '-m "cate_api_dml" -n auto'
+          COVERAGE_PROCESS_START: 'setup.cfg'
```

Comment:
Is there a problem with running these all together instead of one after the other?  If not, you could use `-m "cate_api or cate_api_dml"` in a single call to pytest.

Reply:
Makes sense.